### ISSUE TITLE
IntersectionObserver constructor error: Infinity in threshold array

### ISSUE THREAD

**Issue by bfreuden** [BKWLD/vue-in-viewport-mixin#29]
Since it is my first message here: thank you for this great piece of software!

I noticed random (?) IntersectionObserver constructor errors because the threshold array (passed to the constructor) is sometimes containing -Infinity. This is coming from a division by target.height that equals 0 (see attached screenshots below,  with various debugger watches).
Honestly I don't know how I can help any further but if you can suggest me additional investigations, I would be happy to be "remote controlled" :) 

![mixin error 1](https://user-images.githubusercontent.com/5137152/91975515-cc904f00-ed1f-11ea-83fb-dae4495e4fd8.png)

![mixin error 2](https://user-images.githubusercontent.com/5137152/91975535-d1550300-ed1f-11ea-8207-c27c613cf7de.png)

**Comment by bfreuden** [comment]
If I do something like this in the coffee script I don't get the error anymore.
But I can't tell if I'm simply adding a workaround to a more serious root problem.

```
			maxThreshold = Math.min 1, root.height / target.height
			if not isFinite maxThreshold
				maxThreshold = 1
			@inViewport.maxThreshold = maxThreshold

```
At first sight my app seem to behave normally now (that error had an odd side effect because inViewport values were incorrect).

**Comment by can-keklik** [comment]
Probably duplicate of https://github.com/BKWLD/vue-in-viewport-mixin/issues/20

**Comment by weotch** [comment]
Closed by #20
